index.html

```diff
@@ -35,83 +46,956 @@
+            </ul>
+
+            <section>
+                <h4>Due to Content Security Policy (CSP)</h4><p>
+                    Use of some JavaScript APIs such as <code>eval()</code> are restricted due to CSP. If an explicit policy is not specified, the following default policy is applied.
+                    <ul>
+                        <li>Script execution via <code>eval()</code> and inline script execution are not allowed</li>
+                        <li>Only script from the extension's package can be executed</li>
+                        <li>Executing script from the web is not allowed</li>
+                        <li>Content scripts are only allowed to inject script that executes immediately. Injected script that will only execute after an event-based delay is not permitted. Instead, content script must be used to handle non-immediate events.</li>
+                    </ul>
+                </p>
+            </section>
+
+            <section>
+                <h4>Due to Execution Context</h4>
+                Browser extension pages and their corresponding scripts are executed in one of the following contexts:
+                <ul>
+                    <li>
+                        Browser extension contexts ("Window") - Most extension APIs are available to this context
+                        <ul>
+                            <li>Background page - use the "background" manifest key and specify "persistent" true </li>
+                            <li>Event page - use the "background" manifest key and specify "persistent" false </li>
+                            <li>Popup page - use the "browser_action" or "page_action" manifest key and specify "detault_popup" </li>
+                            <li>Options page - use the "options_page" manifest key </li>
+                            <li>Custom page - such as "browserext:&#x2F;&#x2F;&lt;ext_id&gt;&#x2F;myCustomPage.html" </li>
+                        </ul>
+                    </li>
+                    <li>Content script context - A limited number of APIs are available to script that extensions inject into hosted web site pages
+                    <li>Web page context - Messaging APIs can be used to send messages from hosted web site pages such as https://en.wikipedia.org/wiki/Main_Page/index.html or https://w3c.github.io/html/index.html to extensions
+                </ul>
+            </section>
+
+            <section>
+                <h4>Due to Manifest Keys and Permissions</h4>
+                Some browser extension API require a specific manifest key or specific permission. If the key or permission are not declared or not granted, the object will not be available for scripting. These requirements are summarized in the table below.<br /><br />
+                <font style="background:yellow">DISCUSS: Should we remove the "Optional Permissions" column? Do we plan to support this in the common/core spec?</font><br/>
```

Comment:
Please use `<span class="...">` rather than `<font>` tags, and `<p>` rather than `<br>`